buildSrc/src/main/kotlin/otel.publish-conventions.gradle.kts

```diff
@@ -79,7 +79,7 @@ afterEvaluate {
 
 fun computeArtifactId(): String {
     val path = project.path
-    if (!path.contains("auto-instrumentation")) {
+    if (!path.contains("instrumentation")) {
```

Comment:
This change is not needed to initialize services, although I had to do some local tests and noticed that the instrumentation artifacts were not found due to their recent name change. So this change makes the instrumentation artifact's names stay the same from before the "auto-instrumentation" module was renamed to "instrumentation".

Reply:
I'd probably make this type-safe, a list of modules with their types, so you can check if the artifact here is part of the list (if possible), maybe you don't have access to this info here, just throwing an idea.

Reply:
@LikeTheSalad Great catch! Thank you.

@marandaneto I appreciate where you are coming from. I have opened this issue to track this: https://github.com/open-telemetry/opentelemetry-android/issues/273

Reply:
> I'd probably make this type-safe, a list of modules with their types, so you can check if the artifact here is part of the list (if possible), maybe you don't have access to this info here, just throwing an idea.

I'm not sure if we have that info available but I'm open to ideas! I was planning to talk about this in today's Android SIG to get more details on it 😅

Reply:
We commented at the same time, heh. I made an issue to track this.